src/pages/Login.js

```diff
@@ -26,6 +32,7 @@ export default function Login() {
           />
         </Container>
       </div>
+      {error ? <Alert variant="danger">{error}</Alert> : <></>}
```

Comment:
Two quick things - you can do just `</>`, or better yet, `null` to avoid creating an extra component.